src/Illuminate/Support/helpers.php

```diff
@@ -1170,3 +1170,17 @@ function with($object)
         return $object;
     }
 }
+
+if (! function_exists('null_if')) {
+    /**
+     * Return null if $condition is true, otherwise return a value.
```

Comment:
Change `$condition` to just `condition`!

Reply:
Thanks for noticing and fixed